Package.swift

```diff
@@ -1,53 +1,40 @@
-// swift-tools-version:5.2
+// swift-tools-version:5.3
 
 import Foundation
 import PackageDescription
 
-#if swift(>=5.3)
-let filePath = #filePath
-#else
-let filePath = #file
-#endif
-
-let isOss = !FileManager.default.fileExists(
-  atPath: URL(fileURLWithPath: filePath)
-    .deletingLastPathComponent()
-    .appendingPathComponent("Sources")
-    .appendingPathComponent("Models")
-    .appendingPathComponent("Transcripts")
-    .appendingPathComponent(".git")
-    .path
-)
-
-extension SwiftSetting {
-  static let warnLongExpressionTypeChecking = unsafeFlags(
-    [
-      "-Xfrontend", "-warn-long-expression-type-checking=200",
-      "-Xfrontend", "-warn-long-function-bodies=200",
-    ],
-    .when(configuration: .debug)
-  )
-}
-
-extension Array where Element == SwiftSetting {
-  static let pointFreeSettings: Array = isOss
-    ? [.define("OSS"), .warnLongExpressionTypeChecking]
-    : [.warnLongExpressionTypeChecking]
-}
-
-let package = Package(
+var package = Package(
   name: "PointFree",
   platforms: [
-    .macOS(.v10_15),
+    .macOS(.v11),
   ],
   products: [
     .executable(name: "Runner", targets: ["Runner"]),
     .executable(name: "Server", targets: ["Server"]),
+    .library(name: "Database", targets: ["Database"]),
+    .library(name: "DatabaseTestSupport", targets: ["DatabaseTestSupport"]),
+    .library(name: "DecodableRequest", targets: ["DecodableRequest"]),
+    .library(name: "EmailAddress", targets: ["EmailAddress"]),
+    .library(name: "FoundationPrelude", targets: ["FoundationPrelude"]),
+    .library(name: "FunctionalCss", targets: ["FunctionalCss"]),
+    .library(name: "GitHub", targets: ["GitHub"]),
+    .library(name: "GitHubTestSupport", targets: ["GitHubTestSupport"]),
+    .library(name: "Mailgun", targets: ["Mailgun"]),
+    .library(name: "Models", targets: ["Models"]),
+    .library(name: "ModelsTestSupport", targets: ["ModelsTestSupport"]),
+    .library(name: "PointFree", targets: ["PointFree"]),
+    .library(name: "PointFreePrelude", targets: ["PointFreePrelude"]),
+    .library(name: "PointFreeRouter", targets: ["PointFreeRouter"]),
+    .library(name: "PointFreeTestSupport", targets: ["PointFreeTestSupport"]),
+    .library(name: "Stripe", targets: ["Stripe"]),
+    .library(name: "StripeTestSupport", targets: ["StripeTestSupport"]),
+    .library(name: "Styleguide", targets: ["Styleguide"]),
+    .library(name: "Syndication", targets: ["Syndication"]),
+    .library(name: "Views", targets: ["Views"]),
   ],
   dependencies: [
     .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
     .package(url: "https://github.com/ianpartridge/swift-backtrace.git", .exact("1.1.0")),
-    .package(url: "https://github.com/pointfreeco/Ccmark.git", .branch("main")),
```

Comment:
System packages are deprecated because they can be defined as targets now, as it now is below.